### ISSUE TITLE
Failure to boot with RX 6600 - user error or incompatible GPU

### ISSUE THREAD

**Issue by nikko-bytes** [SchmockLord/Gigabyte-Z590i-Vision-D-11900k#38]
Apologies if this is the wrong forum to raise this problem.
I'm checking to see if this is user error or have a bought the wrong GPU.
I have had the [OC 0.8.3 Preview for macOS Ventura Beta 3](https://github.com/SchmockLord/Gigabyte-Z590i-Vision-D-11900k/releases/tag/v22) working successfully on my Z590i VisionD, RX570 32GB for both 12.5 and Ventura Beta3 and maybe 4. With no changes to the config.plist, drivers or kexts.
Today I bought and attempted to use a new GPU Saphire Pulse RX 6600, instead of very reliable RX 570 and have not managed to get the system to boot.  
I removed the SSDT-BR0-6900XT.aml - no luck kernal panic, see image 
I attempted to revert to 0.7.8 with latest Lilu and Whatevergreen - the boot managed to go somewhat further but got stuck in a boot loop, after trying to load the OS for 20 seconds or so.
Booting into W10, everything works fine.
No changes to BIOS from RX570 to RX660
Open to suggestions or ideas to resolve. It might be something minor I've overlooked.
Thanks
![GPU_kernal panic](https://user-images.githubusercontent.com/73743645/183909486-2edae31a-6af5-48ad-861f-ae7fd7235baf.png)
![Z590i VisionD kernel panic example](https://user-images.githubusercontent.com/73743645/183909492-d2347864-8241-4d1d-b4bb-b796057df7fb.png)

**Comment by nikko-bytes** [comment]
I understand RX6600 is supported from the Dortania guide
It is running with bootarg agpdmod=pikera 

So I should remove this:
			<dict>
				<key>Comment</key>
				<string>PCI-Bridge for RX 6900 XT Card</string>
				<key>Enabled</key>
				<true/>
				<key>Path</key>
				<string>SSDT-BR0-6900XT.aml</string>
			</dict>
and the SSDT-BR0-6900XT.aml

\VEN_**1002**&DEV_**73FF**&SUBSYS_E447**1DA2**&REV_C7\6&16822204&0&00000008 
Device ID - 73FF
Subsystem vendor - 1002
Manufacturer - 1DA2

I've tried resetting the nvram as well.

**Comment by sash11** [comment]
Yes, you should remove the AML file for sure as it is not needed for RX6600.

**Comment by nikko-bytes** [comment]
Update
Success but not 100% of boot attempts. For some reason it boots on the second attempt - strange... This is where I'm currently at:
- removed SSDT-BR0-6900XT and the reference to it in the config.plist (the AMD RX570 worked with this in place)
- added BootArg shikigva=128 - which I didn't try last night
- verbose (-v) to see where things were going off track

On the classic Mac Pro forums, they suggest flashing the BIOS of this card. It was to be the next step if I did not manage to get it working. Glad this wasn't required.

With Device ID = 73FF, where and how would that be entered into config.plist? Should I change this part of the config.plist do you think?
	<key>DeviceProperties</key>
	<dict>
		<key>Add</key>
		<dict>
			<key>PciRoot(0x0)/Pci(0x1,0x0)/Pci(0x0,0x0)/Pci(0x0,0x0)/Pci(0x0,0x0)</key>
			<dict>
				<key>device-id</key>
				<data>v3MAAA==</data>
				<key>shikigva</key>
				<integer>128</integer>

Thanks for the tips.
![Nicks_Z590i_visionD_RX6600](https://user-images.githubusercontent.com/73743645/184055121-756e7dec-e52e-4ec0-a00b-ec5ae8fd8bdc.png)

**Comment by nikko-bytes** [comment]
Update, resolved laggy and slow performance by changing the Device ID to FF731002 for RX6600 GPU. Chris' build Gigabyte-Z590i-Vision-D-11900k-26.
No further action required
![DeviceID that works 25 08 2022](https://user-images.githubusercontent.com/73743645/186671089-10cf8a11-b5db-4e3d-aad5-b337e86fdc3e.png)
.

**Comment by nikko-bytes** [comment]
So far so good with Ventura Beta 6.
![macOS13_Beta6](https://user-images.githubusercontent.com/73743645/187170596-a377e69c-2b3b-462e-9347-0fed862254e4.png)

**Comment by SchmockLord** [comment]
Thanks for sharing this here!
